### DISCUSSION TITLE
how to custom my own language

### DISCUSSION THREAD

**Discussion by MNhat0308** [awcodes/filament-curator#421] [Q&A] [answered]
like the title i want translate it to my own
i have follow this but not work [https://laravel.com/docs/10.x/localization#publishing-the-language-files](https://laravel.com/docs/10.x/localization#publishing-the-language-files)
the path is `lang/vendor/filament-curator/vi/forms.php`

**Comment by awcodes** [comment, marked as answer]
Is your app locale set to vi?

The plugin doesn't have a go translation so as long as your app's locale config is setup properly and you have those files in your lang directory it should be fine. 

But this is a Laravel thing and not a plugin issue.
